relayer/relayer.go

```diff
@@ -129,6 +130,34 @@ func NewRelayer(
 		return nil, nil, err
 	}
 
+	if shouldProcessMissedBlocks {
+		err = processMissedBlocks(logger, &r, sub)
+		if err != nil {
+			logger.Error(
+				"Failed to process historical blocks mined during relayer downtime",
+				zap.Error(err),
+			)
+			return nil, nil, err
+		}
+	} else {
+		err = sub.SetProcessedBlockHeightToLatest()
+		if err != nil {
+			logger.Warn(
+				"Failed to update latest processed block. Continuing to normal relaying operation",
+				zap.String("chainID", r.sourceChainID.String()),
+				zap.Error(err),
+			)
+		}
+	}
+
+	return &r, sub, nil
+}
+
+func processMissedBlocks(
```

Comment:
Can we have `r *Relayer` be a receiver rather than an argument?

Reply:
Thanks for this. Still getting used to being in Go land.  :) Addressed in https://github.com/ava-labs/awm-relayer/pull/89/commits/48f95aa6e25ec99bb654f26ba5ffa861588a2bfc